src/game.cpp

```diff
@@ -4214,7 +4224,7 @@ void Game::updateFrame(ProfilerGraph *graph, RunStats *stats,
 	if (draw_control->range_all) {
 		runData->fog_range = 100000 * BS;
 	} else {
-		runData->fog_range = 0.9 * draw_control->wanted_range * BS;
+		runData->fog_range = draw_control->wanted_range * BS;
```

Comment:
@lhofhansl I based this on #4882, as it modifies exactly the same line.

The intent was to rebase this once #4882 got merged. This is a different change after all.

Rebasing...